client/package.json

```diff
@@ -3,9 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@testing-library/react": "^9.1.4",
```

Comment:
@testing-library/react is better served from devdependencies - You can do this by using `-D` or `--dev`
For yarn that would look like: `yard add -D @testing-library/react`